eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -222,4 +247,93 @@ def fitJWST(eventlabel, s4_meta=None):
                 if meta.isplots_S5 >= 1:
                     lc_model.plot(meta)
 
+            else:
```

Comment:
Can we be more clever here and avoid large if statements that have very similar code blocks???  This is probably worth a longer discussion.

Reply:
See my comments above and elsewhere - I think we can start thinking of fits without shared parameters as just slightly special cases of shared fits which will simplify a lot of the code

Reply:
Done